content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,374 @@
+   forms. In order for an event registration product to be selectable in the :guilabel:`Tickets` tab
+   of an event form, the event registration product type **must** be set to *Event Ticket*.
+
+.. note::
+   To create a new event registration product directly from this field, type in the name of the new
+   event registration product, and select either :guilabel:`Create` or
+   :guilabel:`Create and edit...` from the resulting drop-down.
+
+   Clicking :guilabel:`Create` creates the event registration product, which can be edited later.
+   Clicking :guilabel:`Create and edit...` creates the event registration product, and allows the user
+   to edit its configurations right away on a :guilabel:`Create Product` pop-up window.
+
+.. tip::
+   Existing event registration products can be modified directly from this field, as well, by clicking
+   the :guilabel:`Internal link ➡️ (right arrow)` icon, located beside the event registration product.
+   Doing so reveals that product's page on a separate page. Then, proceed to make any desired
+   modifications.
+
+Add or modify the cost of the ticket in the :guilabel:`Price` field. Then, determine a
+:guilabel:`Sales Start` and :guilabel:`Sales End` date in their respective fields. To do that, click
+into the blank field to reveal a calendar popover. From there, select the desired date and time,
+then click :guilabel:`✔️ Apply`.
+
+Then, if desired, designate a :guilabel:`Maximum` amount of that specific ticket that can be sold.
+
+The :guilabel:`Taken` column will populate with accurate data as tickets are sold.
+
+In the :guilabel:`Color` column, users have the option to add a custom color code to different
+tickets.
+
+If the user wishes to delete any tickets from the :guilabel:`Tickets` tab, click the :guilabel:`🗑️
+(trash can)` icon on the corresponding line for the ticket that should be deleted.
+
+.. tip::
+   To add an optional :guilabel:`Description` column to the :guilabel:`Tickets` tab, click the
+   :guilabel:`(optional columns drop-down)` menu, located to the far-right of the column titles.
+   Then, tick the checkbox beside :guilabel:`Description` from the resulting drop-down menu.
+
+   When added, users have the option to add brief descriptions for each event ticket created, which
+   can be used to inform attendees of any perks or amenities that may coincide with specific tickets.
+
+.. _create_events/event_form/communication:
+
+Communication tab
+-----------------
+
+In the :guilabel:`Communication` tab of an event form, users can create various communications that
+can be scheduled to be sent at specific intervals leading up to, and following, the event.
+
+.. image:: create_events/communication-tab.png
+   :align: center
+   :alt: Typical communication tab on an event form in the Odoo Events application.
+
+.. note::
+   By default, Odoo provides three separate pre-configured communications on every new event form.
```

Comment:
What are the three communication types?

Reply:
I added explanations of each communication in the admonition block, per your suggestion.